schedule/yast/opensuse/agama_playwright.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: agama_playwright
+description: >
+  Playwright test on agama
+schedule:
+  - installation/bootloader_start
+  - yam/agama/auto/playwright_test
```

Comment:
should be under `yam/agama/` as it is not auto-installation.

Reply:
updated